manifests/w/WGlunz/pstoedit/4.2.0.0/WGlunz.pstoedit.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: WGlunz.pstoedit
+PackageVersion: 4.2.0.0
+InstallerType: inno
```

```suggestion
InstallerType: inno
InstallModes:
- interactive
- silent
InstallerSwitches:
  Silent: /verysilent /suppressmsgboxes
```

There was no silent installation switch in previous versions. 🤔